[Task] Add basic usage example for static resolver (#9)
pimcore/static-resolver-bundle: doc/02_Basic_Usage.md

# Basic Usage

> ## Static Resolver

Reviewer: ```suggestion
```

> The basic idea is to replace static function calls with resolver services to make the design more robust and better testable.

Reviewer: ```suggestion
The Static Resolver Bundle allows to replace static function calls with resolver services to make the design more robust and testable:
```

> ### Old usage with static call

Reviewer: ```suggestion
## Old Usage With a Static Call
```

```php

use Pimcore\Tool\Authentication;

final class SecurityService implements SecurityServiceInterface
{
    public function getPimcoreUser(): User
    {
        $pimcoreUser = Authentication::authenticateSession();
        if (!$pimcoreUser instanceof User) {
            throw new Exception('No pimcore user found');
        }
    
        return $pimcoreUser;
    }
}
```

> ### Resolver service injection

Reviewer: ```suggestion
## Replacement With Resolver Service Injection
```
